Sources/Datadog/CrashReporting/Integrations/CrashReportingWithRUMIntegration.swift

```diff
@@ -183,15 +185,17 @@ internal struct CrashReportingWithRUMIntegration: CrashReportingIntegration {
                 url: RUMOffViewEventsHandlingRule.Constants.applicationLaunchViewURL,
                 startDate: crashTimings.realCrashDate,
                 sessionUUID: uuidGenerator.generateUnique(), // create new RUM session
-                crashContext: crashContext
+                crashContext: crashContext,
+                hasReplay: nil // as the crash occured after initializing SDK but before starting the first view, we can't know this
```

Comment:
Relates to our discussion today. This actually sounds like a valid optional flag.

I'm wondering if we can improve this and how harmful for the UX would be defaulting this value to false? 🤔

Reply:
Indeed good catch 👍. I think we should follow the convention in this PR and send `false` instead (+ annotate it in the comment). No chance we could ever figure out anything from `nil`, so `false`  should work good and will make it unified.